src/test/resources/customer-no-key.json

```diff
@@ -36,6 +36,5 @@
   "shippingAddressIds": [],
   "billingAddressIds": [],
   "isEmailVerified": false,
-  "stores": [],
-  "customerGroup": {}
+  "stores": []
```

Comment:
Could not get the reason for the changes here ?

Reply:
This `{}` brackets are considering(file read operation) as null values for the object. So in this case we have one object with all the values null. So, Either the customerGroup should be null or to be removed.